com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -416,6 +424,24 @@ internal set
                 }
             }
 
+            internal bool IsParented
+            {
+                get => GetFlag(k_PositionSlerp);
```

Comment:
Should this be the k_IsParented flag?

Reply:
@samlucas-unity 
Ok, fixed that issue and added a test to validate all flag values (just as a sanity check).
👍